app/controllers/sessions_controller.rb

```diff
@@ -50,6 +52,12 @@ def create_login_record user
       session[:login_id] = login.id
     end
 
+    def create_failure_record user, user_name
+      failure = Failure.create! user: user, user_name: user_name, request: request
+
+      session[:login_id] = failure.id
```

Comment:
```suggestion
      Failure.create! user: user, user_name: user_name, request: request
```
No pondría el tema de guardarlo en el `session` porque no lo vamos a usar. En el otro se guarda para registrar el horario de "cierre", si es que alguna vez hace un logout.

Reply:
perfecto, decis que deje el metodo con esa linea sola o subo esa linea al otro metodo que lo llama?

Reply:
Mejor dejalo como un método por "simetría" con el otro, y por si después le metemos alguna otra lógica.